rtlib/generic_opts.hh

```diff
@@ -99,23 +151,55 @@ class Opts {
 #ifdef LIBRNA_RNALIB_H_
         << " (-[tT] [0-9]+)? (-P PARAM-file)?"
 #endif
-        << " (-[drk] [0-9]+)* (-h)? (INPUT|-f INPUT-file)\n";
-    }
+        << " (-[drk] [0-9]+)* (-h)? (INPUT|-f INPUT-file)\n"
+        << "--help,-H,-h                          print this help message\n"
+#ifdef CHECKPOINTING_INTEGRATED
+        << "--checkpointInterval,-c    d:h:m:s    specify the checkpointing "
+        << "interval, default: 0:0:1:0 (1h)\n"
+        << "--checkpointOutput,-O      PATH       set the path where to store "
+        << "the checkpoints, default: current working directory\n"
+        << "                                      The program will also attempt"
+        << " to read existing checkpoints from a Logfile from this path.\n"
+        << "                                      Use the --checkpointInput "
+        << "option to explictly set the input path for all checkpoints.\n"
+        << "--checkpointInput,-I       PATH       set the path were to read "
+        << "the checkpoints from, default: \n"
+        << "                                      parsed from "
+        << "a checkpoint Logfile located at path specified by the "
+        << "--checkpointOutput option\n"
+        << "                                      Caution: Make sure that the "
+        << "input checkpoints from this path were generated from the same \n"
+        << "                                      command line inputs as the "
+        << "inputs to this binary to ensure the correctness of the answer.\n"
```

Comment:
The `-c` shortcut option for `--checkpointInterval` is actually taken already. I will change it to `-p` to avoid a collision. Thanks for pointing this out.